src/System.Diagnostics.StackTrace/src/System/Diagnostics/SymbolStore/SymAddressKind.cs

```diff
@@ -0,0 +1,40 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Diagnostics.SymbolStore
+{
+    [Serializable]
```

Comment:
Does .NET Core need the serializable attributes? (I have no idea, I am just wondering)